tests/test_high_level.py

```diff
@@ -196,12 +196,14 @@ def test_loading_undownloaded(self, tmp_path):
 class TestGetDatasetMetadata:
     "Test high-level get_dataset_metadata function."
 
-    def test_human_param(self, loaded_schemata):
+    def test_human_param(self):
+        "Test the ``human`` parameter of ``get_dataset_metadata``."
+
         name, version = 'gmb', '1.0.2'
 
         gmb_metadata = get_dataset_metadata(name, version=version)
-        dataset_schema = loaded_schemata.schemata['datasets'].export_schema('datasets', name, version)
-        license_schema = loaded_schemata.schemata['licenses'].export_schema('licenses')
+        dataset_schema = _get_schemata().schemata['datasets'].export_schema('datasets', name, version)
```

Comment:
Why are we changing from `loaded_schemata` to `_get_schemata` here?

Reply:
`loaded_schemata` is different from `_get_schemata`, they just happened to be the same before. See my additional comments under `loaded_schemata`.

Reply:
I changed these to `export_schemata()`

Reply:
ah ok I see now, thanks!